app/helpers/application_helper/toolbar/x_vm_cloud_center.rb

```diff
@@ -128,15 +128,36 @@ class ApplicationHelper::Toolbar::XVmCloudCenter < ApplicationHelper::Toolbar::B
     ),
   ])
   include ApplicationHelper::Toolbar::Cloud::InstanceOperationsButtonGroupMixin
-  button_group('vm_access', [
-    button(
-      :cockpit_console,
-      'pficon pficon-screen fa-lg',
-      N_('Open a new browser window with Cockpit for this VM.  This requires that Cockpit is pre-configured on the VM.'),
-      N_('Web Console'),
-      # :image   => "cockpit",
-      :url   => "launch_cockpit",
-      :klass => ApplicationHelper::Button::CockpitConsole
-    )
-  ])
+  button_group(
+    'vm_access',
+    [
+      select(
+        :vm_remote_access_choice,
+        'fa pficon-screen fa-lg',
+        N_('VM Remote Access'),
+        N_('Access'),
+        :items => [
+          # TODO: remove cockpit button because backend isn't implemented yet.
+          button(
+            :cockpit_console,
```

Comment:
The order of the buttons here should be consistent with the other screens. So `VM console` first and `Web Console` as the second.